src/utils/vaultMath.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  floorDivideBy,
+  floorMultiplyBy,
+} from '@agoric/zoe/src/contractSupport';
+import { AmountMath } from '@agoric/ertp';
+import { Ratio } from 'store/vaults';
+
+export const computeToReceive = (
```

Comment:
up for including some unit tests?

do you think these utils would be worthwhile to offer from `@agoric/inter-protocol/tools`?

Reply:
I realized I also need to take into account the loan fee, which seems to shave off a percentage of your value locked, reducing your overall collateralization ratio. I'm not entirely sure how to calculate it though. It definitely makes sense to put these helpers in `inter-protocol` because then we could test creating loans with the amounts generated.

Reply:
After discussion I think the loan fee incorporation is just a nice-to-have. It won't cause offer safety or vault creation to fail, but might affect overall UX, but I think that needs more thought from a product standpoint.

Reply:
ok let's come back to it later